src/utils.js

```diff
@@ -112,7 +97,7 @@ exports.getPathToNodeModules = () => {
 /**
  * Get the config for Listr.
  *
- * @returns {Object}
+ * @returns {object}
```

Comment:
Why has `Object` changed to `object`?  Is `Object` not the type?

Reply:
It's the linter default https://github.com/gajus/eslint-plugin-jsdoc#why-not-capital-case-everything